src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -338,6 +320,7 @@ public WebView(WebView webView) : this(Interop.WebView.NewWebView(WebView.getCPt
         /// <summary>
         /// The url to load.
         /// </summary>
+        /// <since_tizen> 9 </since_tizen>
         [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Please keep [EditorBrowsable(EditorBrowsableState.Never)] in DevelNUI branch.
When this patch is merged, you can remove it on Master branch with ACR.